test.js

```diff
@@ -49,6 +49,35 @@ function describeTestsWithOptions(options, postText) {
       const res = utils.getAttrs(replaceDelimiters(src, options), 0, options);
       assert.deepEqual(res, expected);
     });
+
+    it(replaceDelimiters('should parse attributes whose are ignored the key chars(\\t,\\n,\\f,\\s,/,>,",\',=) eg: {gt>=true slash/=trace i\\td "q\\fnu e\'r\\ny"=}', options), () => {
+      const src = '{gt>=true slash/=trace i\td "q\fu\ne\'r\ny"=}';
+      const expected = [['gt', 'true'], ['slash', 'trace'], ['id', ''], ['query', '']];
+      const res = utils.getAttrs(replaceDelimiters(src, options), 0, options);
+      assert.deepEqual(res, expected);
+    });
+
+    it(replaceDelimiters('should throw an error while calling `hasDelimiters` with an invalid `where` param', options), () => {
+      assert.throws(() => utils.hasDelimiters(0, options), { name: 'Error', message: /Should be "start", "end" or "only"/ });
+      assert.throws(() => utils.hasDelimiters('', options), { name: 'Error', message: /Should be "start", "end" or "only"/ });
+      assert.throws(() => utils.hasDelimiters(null, options), { name: 'Error', message: /Should be "start", "end" or "only"/ });
+      assert.throws(() => utils.hasDelimiters(undefined, options), { name: 'Error', message: /Should be "start", "end" or "only"/ });
+      assert.throws(() => utils.hasDelimiters('center', options)('has {#test} delimiters'), { name: 'Error', message: /expected 'start', 'end' or 'only'/ });
+    });
+
+    it('should escape html entities(&,<,>,") eg: <a href="?a&b">TOC</a>', () => {
```

Comment:
I do not think this project to the escaping? Guess this is tested for in markdown-it?

Reply:
Yes, I agree with you. This function `escapeHtml` wasn't used in this project. But It is `exports` for outer usage. This test case is only for coverd L274-277.